ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-ocp4.14-lp-rosa.yaml

```diff
@@ -0,0 +1,66 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: maistra-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: servicemesh-rosa-aws
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
```

Comment:
Can we use the firewatch v2 syntax for this firewatch config?

This will allow us to specify jira_additonal_labels in a single place instead of in every line among some other things.

Here's Caleb PR where he introduced the new firewatch syntax to our self-managed scenarios
https://github.com/openshift/release/pull/46282

Reply:
Yes @ascerra , the PR is updated with the latest commit to use the firewatch v2 syntax.